src/js/pensions/helpers.jsx

```diff
@@ -104,7 +104,7 @@ export const directDepositWarning = (
   </div>
 );
 
-export const applicantDescription = <p>You aren’t required to fill in <strong>all</strong> fields, but VA can evaluate your claim faster if you provide more information.</p>;
+export const applicantDescription = <p>You aren’t required to fill in <strong>all</strong> fields, but we can review your application faster if you provide more information.</p>;
```

Comment:
Same here, can we pull in the common component?

Reply:
Thanks @jbalboni, excellent idea, change made.